phpcs.xml.dist

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0"?>
+<ruleset name="WP-CLI-i18n">
+	<description>Custom ruleset for WP-CLI I18n-command</description>
+
+	<!--
+	#############################################################################
+	COMMAND LINE ARGUMENTS
+	For help understanding this file: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Annotated-ruleset.xml
+	For help using PHPCS: https://github.com/squizlabs/PHP_CodeSniffer/wiki/Usage
+	#############################################################################
+	-->
+
+	<!-- What to scan. -->
+	<file>.</file>
+
+	<!-- Ignoring select files/folders.
+		 https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-files-and-folders -->
+	<exclude-pattern>*/tests/data/*</exclude-pattern>
+
+	<!-- Show progress. -->
+	<arg value="p"/>
+
+	<!-- Strip the filepaths down to the relevant bit. -->
+	<arg name="basepath" value="./"/>
+
+	<!-- Check up to 8 files simultaneously. -->
+	<arg name="parallel" value="8"/>
+
+	<!--
+	#############################################################################
+	USE THE WP_CLI_CS RULESET
+	#############################################################################
+	-->
+
+	<rule ref="WP_CLI_CS"/>
+
+	<!--
+	#############################################################################
+	PROJECT SPECIFIC CONFIGURATION FOR SNIFFS
+	#############################################################################
+	-->
+
+	<!-- For help understanding the `testVersion` configuration setting:
+		 https://github.com/PHPCompatibility/PHPCompatibility#sniffing-your-code-for-compatibility-with-specific-php-versions -->
+	<config name="testVersion" value="5.4-"/>
+
+	<!-- Verify that everything in the global namespace is either namespaced or prefixed.
+		 See: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/wiki/Customizable-sniff-properties#naming-conventions-prefix-everything-in-the-global-namespace -->
+	<rule ref="WordPress.NamingConventions.PrefixAllGlobals">
+		<properties>
+			<property name="prefixes" type="array">
+				<element value="WP_CLI\I18n"/><!-- Namespaces. -->
+				<element value="wpcli_i18n"/><!-- Global variables and such. -->
+			</property>
+		</properties>
+	</rule>
+
+	<!-- Whitelist method names for classes that extend a PSR-2 package. -->
+	<rule ref="WordPress.NamingConventions.ValidFunctionName.MethodNameInvalid">
+		<exclude-pattern>*/src/IterableCodeExtractor\.php$</exclude-pattern>
+	</rule>
+
+	<!-- Whitelist variable names for classes that extend a PSR-2 package. -->
```

Comment:
It would be better to use the specific configuration for this sniff to allow for these.
See: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/wiki/Customizable-sniff-properties#mixed-case-property-name-exceptions

Reply:
@jrfnl There was actually three different ones that were triggered, so I didn't bother listing them individually anymore.

Reply:
The problem with that is that now newly introduced variable name violations in these files will no longer be reported, which is why I suggested using the property.

Reply:
Oh, I'm sorry, I should have actually read the link you posted. So instead of whitelisting against the rule, I should whitelist against the individual variable names.

Yes, I agree that that makes more sense, I'll push a subsequent PR.

Reply:
Hmm, that only seems to work for properties, not variables. Am I missing a specific notation?
Here's what seems to work for the properties:
<img width="611" alt="Image 2019-04-24 at 5 57 28 PM" src="https://user-images.githubusercontent.com/83631/56674498-75ee1c00-66ba-11e9-9927-cb5f0ed9f114.png">
This leaves me with the following errors:
<img width="594" alt="Image 2019-04-24 at 5 58 10 PM" src="https://user-images.githubusercontent.com/83631/56674547-8acaaf80-66ba-11e9-9cab-ea4664cf7e72.png">

Reply:
Having a look now at the files in question. This is what I see:

1. `src/JSFunctionsScanner.php`: I see no reason why the `$extractComments` property hasn't been renamed. It is a `private` property, so independent of the parent class. Why should this not comply ?
2. `src/MapCodeExtractor.php`: `$mapObject` is a function local variable, so it is perfectly safe to rename it. Why should this not comply ?
3. `src/MapCodeExtractor.php`: `sourcesContent` is a property and can be whitelisted via the above.
4. `src/PhpFunctionsScanner.php`: `$extractedComment` is a function local variable, so it is perfectly safe to rename it. Why should this not comply ?

So basically this would be the ruleset config. Everything else should just be renamed:
```xml
	<rule ref="WordPress.NamingConventions.ValidVariableName">
		<properties>
			<property name="customPropertiesWhitelist" type="array">
				<element value="sourcesContent"/>
			</property>
		</properties>
	</rule>

```

Reply:
You're right, I did these changes in https://github.com/wp-cli/i18n-command/pull/161